ui/Screen/Settings.svelte

```diff
@@ -0,0 +1,144 @@
+<script>
+  import { Title, Text, Button } from "../DesignSystem/Primitive";
+  import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
+
+  const themeOptions = [
+    {
+      title: "Light",
+      value: 0,
+    },
+    {
+      title: "Dark",
+      value: 1,
+    },
+  ];
+  const networkOptions = [
+    {
+      title: "Testnet",
+      value: 0,
+    },
+    {
+      title: "Mainnet",
+      value: 1,
+    },
+  ];
+</script>
+
+<style>
+  .container {
+    max-width: 71.25rem;
+    margin: 0 auto;
+    min-width: 640px;
+  }
+  .section-header {
```

Comment:
Can we use html tags for this instead? i.e. `section header`.

Reply:
:+1: